wizard/src/utils/detect_place.ts

```diff
@@ -28,52 +86,61 @@ export class PlaceDetector {
   countryAbbrv3: Set<string>;
   countryNumeric: Set<string>;
 
-  placeTypes: Map<string, string>;
+  // A set of all place types and their associated properties.
+  placeTypesAndProperties: Set<TypeProperty>;
 
-  static typeFormatMappings = new Map<string, Array<DetectedFormat>>([
-    ["Longitude", [{ propertyName: "longitude", displayName: "Longitude" }]],
-    ["Latitude", [{ propertyName: "latitude", displayName: "Latitude" }]],
+  // Mapping between Place types and supported properties associated with each
+  // type.
```

Comment:
Maybe I'm missing something, but not sure why we need the key indirection, and why this cannot just be a map from type as key and list of properties as values.

Reply:
As discussed offline, the keys here are to match the user defined column names.